modules/govuk/files/etc/govuk/import_mapit_data.sh

```diff
@@ -17,11 +17,24 @@ function restart_mapit_services () {
   sudo service nginx start
 }
 
-stop_mapit_services
-# This script needs to run as a user that can sudo to become the postgres
-# user.  It's easiest if this is passwordless, so we can't use the deploy user
-# who can't sudo without a password
-# Run 'govuk_setenv mapit ./import-db-from-s3.sh' from the '/var/apps/mapit' as
 
-cd /var/apps/mapit && govuk_setenv mapit ./import-db-from-s3.sh
+function import_db_from_s3 () {
+  # get the data from s3
+  curl 'https://s3-eu-west-1.amazonaws.com/govuk-custom-formats-mapit-storage-production/source-data/2020-11/mapit-november-2020-update.sql.gz' -o /tmp/mapit.sql.gz
+  if ! echo "e6d7d0049348744cf4c8c2f3f921adfcc0a3c76d /tmp/mapit.sql.gz" | sha1sum -c -; then
+    echo "SHA1 does not match downloaded file!"
+    exit 1
+  fi
+
+  # drop the old db and bring up a new empty one
+  sudo -u postgres dropdb --if-exists mapit
+  sudo -u postgres createdb --owner mapit mapit
+  sudo -u postgres psql -c "CREATE EXTENSION postgis; CREATE EXTENSION postgis_topology;" mapit
+
+  # load up the downloaded data into this new db
+  zcat -f /tmp/mapit.sql.gz | sudo -u postgres psql mapit
```

Comment:
Can we remove the `-f`? I don't think there should be any need to use the "force" option here.

Reply:
I will try, to be honest, I copied the script from Mapit without looking into the specifics.